src/context/worktray-context.tsx

```diff
@@ -296,6 +298,7 @@ export const WorktrayURLProvider: FC<{
         pageSize,
         timePeriod,
         patch,
+        patchId,
```

Comment:
With current implementation the patchId seems to be useless. Either remove patch and rename it with patchId, or just use patch.